src/providers/commandsProvider.ts

```diff
@@ -94,9 +94,6 @@ function execCommand(currentRedPath: string, command: string, fileUri?: vscode.U
         filePath = fileUri.fsPath;
     }
 
-    if (filePath.indexOf(' ') > 0) {
```

Comment:
@geekyi I had originally done that. But it was giving some issues which i don't remember now 😄.